core/shared/src/test/scala/zio/prelude/fx/ZPureSpec.scala

```diff
@@ -842,35 +842,7 @@ object ZPureSpec extends DefaultRunnableSpec {
               )
             }
           )
-        ),
-        test("parallel errors example") {
```

Comment:
Do these tests even need to be removed? I'm fine with keeping the current functionality in ZPure :wink:

Reply:
Good point! Added them back.